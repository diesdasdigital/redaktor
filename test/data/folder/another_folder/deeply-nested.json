{
  "default": {
    "required": {
      "title": "Deeply Nested",
      "pageType": "code-preview"
    },
    "fields": {},
    "components": []
  }
}
