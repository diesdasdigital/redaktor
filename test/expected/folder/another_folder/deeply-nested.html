<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Deeply Nested</title>
  <meta name="viewport" content="width=device-width,initial-scale=1" />
</head>
<body>
  <pre>{
  "data": {
    "default": {
      "required": {
        "title": "Deeply Nested",
        "pageType": "code-preview"
      },
      "fields": {},
      "components": []
    }
  },
  "path": "folder/another_folder/deeply-nested"
}</pre>
</body>
</html>